src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1845,11 +1845,10 @@ protected override void InvokeDefaultAction(string path)
                                 //  a) the user has asked to with the -FollowSymLinks switch parameter and
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Get-ChildItem.FileSystem provider.Understand APPEXECLINKs
```powershell
failed here
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Get-ChildItem.Tests.ps1: line 180
180:             throw "failed here"

```